packages/adaptivecpp/package.py

```diff
@@ -140,8 +187,33 @@ class Adaptivecpp(CMakePackage):
         " Choose one of +nvcxx or +cuda +omp_llvm.",
     )
 
+    # Spack doesn't seem to populate the spec with the default multivalued
+    # variant information.
```

Comment:
Is it worth raising an issue about this upstream?

Reply:
Probably, it seems like a bug.